Sources/MuxPlayerSwift/Fairplay/FairplaySessionManager.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  FairplaySessionManager.swift
+//
+//
+//  Created by Emily Dixon on 4/19/24.
+//
+
+import Foundation
+import AVFoundation
+
+class FairplaySessionManager {
+    
+    static let shared = FairplaySessionManager()
```

Comment:
This should be exposed as a property on `PlayerSDK`, this keeps the number of singletons at one and prevents races during initialization.